<launch>
  <arg name="name" default="raspicam_node" />
  <node type="raspicam_node" pkg="raspicam_node" name="$(arg name)" output="screen">
 
    <param name="camera_info_url" value="package://$(arg name)/camera_info/camera_module_v2_640x480.yaml"/>
    <param name="width" value="640"/>
    <param name="height" value="480"/>
    <param name="framerate" value="30"/>
    <param name="camera_frame_id" value="raspicam"/>

  </node>
  
  <node pkg="rosservice" type="rosservice" name="start_capture" args="call --wait $(arg name)/start_capture"/>

</launch>
